include/ebpf_program_attach_type_guids.h

```diff
@@ -31,29 +31,41 @@ extern "C"
     __declspec(selectany) ebpf_attach_type_t EBPF_ATTACH_TYPE_BIND = {
         0xb9707e04, 0x8127, 0x4c72, {0x83, 0x3e, 0x05, 0xb1, 0xfb, 0x43, 0x94, 0x96}};
 
-    /** @brief The programs attached to the INETx_CONNECT hook will be invoked for
-     * connect() calls on TCP or UDP sockets or when a UDP socket calls sendto() to
+    /** @brief The programs attached to the INET4_CONNECT hook will be invoked for
+     * connect() calls on TCP or UDP sockets or when a UDP socket sends a packet to
      * a unique remote address/port tuple. May be scoped to a network compartment. The programs return 1 to permit a
```

Comment:
I had removed the last 2 sentences in #891. As now this is called out in documentation of the new hook type.

Reply:
Rebased